homeassistant/components/climate/ecobee.py

```diff
@@ -22,16 +22,23 @@
 _LOGGER = logging.getLogger(__name__)
 
 ATTR_FAN_MIN_ON_TIME = 'fan_min_on_time'
+ATTR_RESUME_ALL = 'resume_all'
 
 DEPENDENCIES = ['ecobee']
 
 SERVICE_SET_FAN_MIN_ON_TIME = 'ecobee_set_fan_min_on_time'
+SERVICE_RESUME_PROGRAM = 'ecobee_resume_program'
 
 SET_FAN_MIN_ON_TIME_SCHEMA = vol.Schema({
     vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
     vol.Required(ATTR_FAN_MIN_ON_TIME): vol.Coerce(int),
 })
 
+RESUME_PROGRAM_SCHEMA = vol.Schema({
+    vol.Optional(ATTR_ENTITY_ID): cv.entity_ids,
+    vol.Optional(ATTR_RESUME_ALL): cv.boolean,
```

Comment:
Thanks for the review, Martin.  I made the change so the default value is in the schema.

Regarding making refresh_all a required argument, I don't think it should be required because it is not a mode or state that the thermostat is having set on it.  For example, `set_away_mode` and `set_operational_mode` both take a mode which is then applied to the device.  Same with `set_temperature` which takes an integer to apply to the device.  On the other hand, `refresh_all` is not applied to the device, and only modifies the behavior of the service, so I don't think it should be required.

Let me know if you feel differently.  Thanks again!